racetrack_commons/racetrack_commons/entities/dto.py

```diff
@@ -38,6 +39,7 @@ class JobDto(BaseModel, arbitrary_types_allowed=True):
     deployed_by: Optional[str] = None
     last_call_time: Optional[int] = None
     infrastructure_target: Optional[str] = None
+    replica_internal_names: list[str] = []
```

Comment:
The purpose of this field is quite cryptic, adding comment what is its purpose would be useful (without knowing about this MR, I would have thought it's for some internal scaled pod handling, for sure not for something like grafana/prometheus).